### PR TITLE
setup minimal book from docs

### PR DESCRIPTION
**Pull request by Gankra**
fixe #190

### CODE REVIEW

File: .github/workflows/web.yml [hunk 1]
```diff
@@ -1,32 +1,101 @@
...
+      - completed
 
+# Alright, let's do it!
 jobs:
   web:
     name: Build and deploy site and docs
     runs-on: ubuntu-latest
     steps:
+      # Setup
       - uses: actions/checkout@v2
```
Comment (ashleygwilliams) [inline .github/workflows/web.yml]:
```suggestion
      - uses: actions/checkout@v3
```

Comment (ashleygwilliams) [reply .github/workflows/web.yml]: i updated the other workflow but missed this one i guess

File: docs/book.toml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[book]
+authors = ["Aria Beingessner"]
```
Comment (ashleygwilliams) [inline docs/book.toml]:
```suggestion
authors = ["axodotdev"]
```

File: docs/src/SUMMARY.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Summary
+
+- [Introduction](./introduction.md)
+- [Install](./install.md)
```
Comment (ashleygwilliams) [inline docs/src/SUMMARY.md:4]: should we drop this?

**Review by ashleygwilliams — CHANGES_REQUESTED** [review]
a few small suggestions- also for oranda to pick this up, we'll need to update the oranda.json

File: docs/src/SUMMARY.md (hunk 3 above)
Comment (Gankra) [reply docs/src/SUMMARY.md:4]: it's discardable 100% but we should at very least link oranda's own generated stuff *from* the book with the heading "install" so people can find it quickly if they end up here

Comment (ashleygwilliams) [reply docs/src/SUMMARY.md:4]: we'll handle book updates post-release, since this is not a release we are formally announcing

**Review by ashleygwilliams — APPROVED** [review]

**Pull request merged**